### ISSUE TITLE
disable integration of Handle server and related dependencies by default

### ISSUE THREAD

**Issue by saschaszott** [DSpace/DSpace#9905]
Add a Maven profile (which should be disabled by default) to control the inclusion of all runtime dependencies related to the Handle server in DSpace.

Is it possible to move the Handle server related Java classes in package `org.dspace.handle` (https://github.com/DSpace/DSpace/tree/main/dspace-api/src/main/java/org/dspace/handle) in a separate Maven module?

**Comment by saschaszott** [comment]
There was a discussion in Slack:

@mwoodiupui
> I have an alternate proposal:  DSpace should be an ordinary client of the stock Handle server through the Handle network protocol instead of tightly integrating with it through the database.  That might considerably reduce the amount of "foreign" code in DSpace related to Handles, and it decouples DSpace from Handle server version considerations.  I've been wanting to write this for years, but have never found a satisfactory answer to:  how to set up a "feral" Handle server not connected to the master server, without violating the terms of the license or HSA.

@tdonohue :
> FWIW, I'd love that. Still, a Maven profile is a "simple fix" while that larger refactor is being worked on.  But, yes, I'd much rather DSpace didn't bundle all the Handle stuff.  But, we'd have to find a way to make that all work & document how to connect DSpace to a Handle Server properly, etc.

**Comment by tdonohue** [comment]
Agreed in concept with the idea. But, it needs a volunteer to investigate how to achieve this.  I'm not sure myself how easy this will be to achieve, or even how best to achieve it.
